packages/lndr/format/index.ts

```diff
@@ -167,23 +164,8 @@ export const nickLengthIncorrect = nick => typeof nick === 'string' && nick.leng
 
 export const bcptAmount = amount => amount.replace(/[^0-9]/g, '')
 
-function convertToCommaDecimal(amount: string) : string {
-  if(amount.match(/\.[0-9]{2}$/) !== null) {
-    const end = amount.slice(-2)
-
-    return amount.slice(0, -3)
-    .replace(/,/, '.')
-    .concat(',').concat(end)
-  } else if(amount.match(/\.[0-9]{1}$/) !== null) {
-    const end = amount.slice(-1)
-
-    return amount.slice(0, -2).replace(/,/, '.').concat(',').concat(end)
-  }
-  return amount
-}
-
-export const convertCommaDecimalToPoint = (amount: string, currency: string) : string => {
-  if(!isCommaDecimal(currency)) {
+export const convertCommaDecimalToPoint = (amount: string) : string => {
```

Comment:
Removing the second parameter here is fine, but it was not removed from the caller in `/ui/dialogs/settlement/index.tsx` on line 209

`209     const formattedAmount = convertCommaDecimalToPoint(amount, primaryCurrency)`